modules/nodes-pods-autoscaling-creating-web-console.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-about.adoc
+
+[id="nodes-pods-autoscaling-creating-web-console_{context}"]
+
+= Creating a horizontal pod autoscaler by using web console
+
+From the web console, you can create a horizontal pod autoscaler that specifies the minimum and maximum number of pods you want to run on a deployment. You can also define the amount of CPU or memory use that your pods should target..
+
+[NOTE]
+====
+Horizontal pod autoscaler cannot be added to those Deployment or Deployment Configs that are part of operator-backed service, Knative service, or Helm chart.
+====
+
+.Procedure
+
+To create a horizontal pod autoscaler in the web console:
+
+. In the *Topology* view, click the node to reveal the side pane.
+. From the *Actions* drop-down list, select the *Add HorizontalPodAutoscaler* to open the *Add HorizontalPodAutoscaler* form.
++
+.Add HorizontalPodAutoscaler
+image::node-add-hpa-action.png[Add HorizontalPodAutoscaler form]
```

Comment:
I do not see the image showing up in the bot preview build, nor do I see it displayed from a local asciibinder build. I think you might need a line break before the next step (`. From the...`).

Reply:
Ok, will try that for sure.

Reply:
Hm, I'm still not seeing the image in the preview build. Perhaps a break between these 2 lines?

```
.Add HorizontalPodAutoscaler
image::node-add-hpa-action.png[Add HorizontalPodAutoscaler form]
```
Not sure if you are checking this issue from a local build, but I suggest seeing if you can get the image to display for you. If not, we can check with the team about what might be causing the issue.

Reply:
Running the build locally causes this error:
`asciidoctor: WARNING: image to embed not found or not readable: /git/review-openshift-docs/openshift-docs/nodes/pods/images/node-add-hpa-action.png`

^^This is because the `nodes/pods/` folder is missing the `modules/` and `images/` symbolic links. From inside the `nodes/pods` folder, you can run the following commands to add the links:

- `ln -s ../../images/ images`
- `ln -s ../../modules/ modules`

Turns out that these are missing in a number of other subdirectories, which @bergerhoffer is addressing in this separate PR: https://github.com/openshift/openshift-docs/pull/31429 (note that she didn't address the file you are working on to avoid any conflicts). Thank you, Andrea!!!

Reply:
I have added the symbolic links of images and modules to the nodes/pods folder. Let's see if the image renders in the preview.

Reply:
Yes, it looks good now: https://deploy-preview-30597--osdocs.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-autoscaling.html#nodes-pods-autoscaling-creating-web-console_nodes-pods-autoscaling